### ISSUE TITLE
Error building android APK

### ISSUE THREAD

**Issue by salahdin-ahmed** [beeware/briefcase#594]
**Describe the bug**
Executing `briefcase build android` results in the following error:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeDebugResources'.
> Multiple task action failures occurred:
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-3.5.3-5435860-linux Daemon #0: Daemon startup failed
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-3.5.3-5435860-linux Daemon #2: Daemon startup failed
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-3.5.3-5435860-linux Daemon #1: Daemon startup failed
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-3.5.3-5435860-linux Daemon #3: Daemon startup failed
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-3.5.3-5435860-linux Daemon #4: Daemon startup failed
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-3.5.3-5435860-linux Daemon #5: Daemon startup failed
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-3.5.3-5435860-linux Daemon #6: Daemon startup failed
        This should not happen under normal circumstances, please file an issue if it does.
   > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
      > AAPT2 aapt2-3.5.3-5435860-linux Daemon #7: Daemon startup failed
        This should not happen under normal circumstances, please file an issue if it does.
```

**Expected behavior**
The app builds perfectly fine and I get an APK that is run-able on any "modern" android device

**Environment:**
 - Operating System: VoidLinux
 - Python version: Python 3.9.6
 - Software versions:
   - Briefcase: 0.3.5
   - Toga:
     - toga==0.2.15
     - toga-core==0.3.0.dev27
     - toga-gtk==0.3.0.dev27
   - OpenJDK8: 8u272b02
   - gradle: 6.8.1
 
**Additional context**
[The app](https://github.com/salahdin-ahmed/sherlock-gui) I am trying to build, although I do not think this would be helpful since I tried building [beeliza](https://github.com/beeware/toga/tree/master/examples/beeliza) and got the same error message

**Comment by freakboy3742** [comment]
Firstly - your list of Toga versions points at a possible problem; `toga==0.2.15` is incompatible with `toga-core` and `toga-gtk` 0.3.0. You're also going to have difficulties if you're trying to use toga-gtk on Android.

However, that's unlikely to be the source of the problem you're reporting here. The error message looks like a fundamental failure of the Android toolchain to start (specifically, Gradle). 

I can only think of two sources for this problem:

1. VoidLinux isn't compatible with the Android toolchain. I know nothing about VoidLinux, so I can't comment on how likely this may be.
2. The tools didn't download correctly. You've got a corrupted or incomplete download, and as a result, gradle isn't starting. If you delete `~/.briefcase/tools` and try `briefcase build` again, that will re-download the pieces that are needed.

**Comment by salahdin-ahmed** [comment]
I found out that the problem is actually with Void Linux, I used [Bedrock Linux](https://bedrocklinux.org/) with a Ubuntu strata and the build went perfectly fine when done completely under Ubuntu. However, the app crashed when ran on Android VM (both the app I am working on and beeliza). Moreover, when tried to install it on an real Android device it also crashed, probably this the problem because of my weird set-up, I will try it in a Ubuntu VM later.
